e2e-tests/playwright/utils/analytics/analytics.ts

```diff
@@ -0,0 +1,34 @@
+import { expect, request } from '@playwright/test';
+
+export class Analytics {
+  async getGuestToken(): Promise<string> {
```

Comment:
@zdrapela can we move this method to `utils/comman.ts`?

Reply:
Makes sense. I moved `getGuestToken` and `getGuestAuthHeader` to `APIHelper`, as they are both related to working with API and can be reused elsewhere. Thanks